src/test/resources/csvwExamples/observations_duplicate_headers.csv

```diff
@@ -0,0 +1,10001 @@
+Geography,Age,Age,Health,Value
```

Comment:
Can we try and make some of these CSV files smaller?

Reply:
Done